shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1204,13 +1206,6 @@ - (void)keyboardWillChangeFrame:(NSNotification*)notification {
     return;
   }
 
-  CGRect keyboardFrame = [[info objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
-  CGRect screenRect = [[UIScreen mainScreen] bounds];
-
-  // Get the animation duration
-  NSTimeInterval duration =
-      [[info objectForKey:UIKeyboardAnimationDurationUserInfoKey] doubleValue];
-
   // Considering the iPad's split keyboard, Flutter needs to check if the keyboard frame is present
   // in the screen to see if the keyboard is visible.
   if (CGRectIntersectsRect(keyboardFrame, screenRect)) {
```

Comment:
This line can be changed to `if (isShowing) {` if you take my suggestion above.

Reply:
I'll fix that this week 👍